### DISCUSSION TITLE
Logout redirects

### DISCUSSION THREAD

**Discussion by AdamTheRedBaron** [oidc-wp/openid-connect-generic#308] [Ideas]
This is a feature request for custom logout redirects. At the moment the logout process finishes at the SSO provider, but ideally I would like to redirect the user back to my site.

**Comment by timnolte** [comment]
I believe this is the same request that in the open Issue https://github.com/oidc-wp/openid-connect-generic/issues/177

**Reply by AdamTheRedBaron** [reply]
@timnolte hi, yes. Sorry for repeating it - I wasn't sure if I made the request in the right place. I'm still a bit of a github newbie.

**Reply by timnolte** [reply]
No worries, we're still navigating the best way to direct people between Issues and Discussion.
